rfay/ddev-drushonhost
# fix: Logic for detecting host-side behavior was broken, add tests

* @stasadev pointed out that the logic originally used (from the beginning of time) was only accidentally working for host-side drush
* Improve the formatting of the actual implementation so it's readable (use HEREDOC)
* Add actual tests to check behavior

## Manual testing:
Please test this with 
```
ddev get https://github.com/rfay/ddev-drushonhost/tarball/20240705_fix_host_side_logic
```

* Install drush on the host side, or add vendor/bin of project to $PATH
* In a Drupal project
* `ddev get https://github.com/rfay/ddev-drushonhost/tarball/20240705_fix_host_side_logic`
* `ddev restart`
* Review the added config.drushonhost.yaml
* `export IS_DDEV_PROJECT=true`
* `ddev mysql` to see if it works
* Do more exotic things like `ddev drush si -y demo_umami`
